turbo.json

```diff
@@ -12,7 +12,7 @@
       "cache": false
     },
     "db-generate": {
-      "cache": false
+      "cache": true 
```

Comment:
How will it know when to invalidate the cache when there's no output set?

Reply:
That's a good question. From what I've gathered from TurboRepo's docs, the hashing process is dependent on more factors than just the `outputs` option specified on the pipeline config (see https://turborepo.org/docs/core-concepts/caching#hashing). 

If i'm to make and educated guess, this is the condition it is relying on in this case:

> Hash the contents of all version-controlled files in the workspace folder or the files matching the inputs globs, if present

So the invalidation will actually happen by checking hash changes between files of the workspace (eg. `index.ts` that exports the Prisma Client, or the actual schema file). This is corroborated by local testing: 

Running `pnpm db-generate` for the first time, no caching:

![image](https://user-images.githubusercontent.com/55156145/197152119-aad9eec7-9786-4751-8234-dc70c24aa8ea.png)

Running `pnpm db-generate` for the second time, without modifying any files on `packages/db`, picks up cached result:

![image](https://user-images.githubusercontent.com/55156145/197152586-4766ddd9-07f5-4261-ac13-68587c48f915.png)

Running `pnpm db-generate` for the third time, now modifying something on the `index.ts` file in `packages/db`, hash does not match, cache is invalidated:

![image](https://user-images.githubusercontent.com/55156145/197153351-ab8c6ce5-b8de-43e6-adea-7886671c65a9.png)